scripts/missions/amk/01_A_Moogle_Kupo_dEtat.lua

```diff
@@ -3,6 +3,10 @@
 -- A Moogle Kupo d'Etat M1
 -- !addmission 10 0
 -----------------------------------
+require('scripts/globals/missions')
```

Comment:
Global requires are no longer necessary outside of dependencies within globals alone (not going to tag every instance)